call-for-papers.md

```diff
@@ -1,54 +0,0 @@
----
-layout: default
-title: Call for Papers
-permalink: /call-for-papers
----
-
-The 16th Annual International Biocuration Conference (Biocuration 2023), supported by the ISB, invites
-attendees to consider two grand challenges for the future of biocuration and the growing biocuration
-community.
-
-First, given the widespread interest in making data, metadata, and knowledge more FAIR
-(Findable, Accessible, Interoperable, Reusable) and TRUST-worthy (Transparency, Responsibility,
-User Focus, Sustainability, and Technology), how can the community move past these as superficial
-platitudes and towards real impact?
-
-Second, given how can the community spread past its prefix of
-"bio-" and into adjacent domains of chemistry, clinical informatics, agriculture, ecology, and related
-natural sciences?
-
-These themes align with our community's core values of openness, transparency,
-collaboration, and reusability on both a philosophical and technical level.
-
-We invite authors to submit abstracts for oral or poster presentation at Biocuration 2023 that highlight
-their recent work in either biocuration or more broadly curation in the life, natural, or computational
-sciences.
-
-Topics for submission include:
-
-- **Generation, annotation, and curation of resources**: Efforts for the curation of data from
```

Comment:
why didn't this nice topic list make it to the new page?

Reply:
Because it is unclear to me whether we want the list. The topics are already mentioned in the text that @monicacecilia shared directly from the CFP on EasyChair.

And should the `abstracts.md` have the list version of topics or should we have it in the submission guide (https://github.com/biocuration2023/biocuration2023.github.io/issues/5).

Open to both ideas. Let me know what makes the most sense.

Reply:
well until someone goes and writes the submission guide, I'd say you should keep it. If you restore that text, we can merge this PR then move on, and someone else can thinking about if they want to move things. Let's try and keep things one at a time rather than playing 3D chess about what goes where :)

Reply:
Sounds good. We can always restructure it later on!

Pushed the changes with the list of topics as well. Feel free to merge if you think its ready 👍